runtime/src/bank.rs

```diff
@@ -6322,31 +6322,49 @@ impl Bank {
     }
 }
 
-/// Scan all the accounts for this bank and collect stats
+/// Get all the accounts for this bank and calculate stats
 pub fn get_total_accounts_stats(bank: &Bank) -> ScanResult<TotalAccountsStats> {
+    // This implementation could have used scan_accounts() directly, but does not so that it could
+    // share `calculate_total_accounts_stats()` with ledger-tool (see subcommand "accounts").
```

Comment:
I think scrap this comment, as it's clear from the implementation and callers.

Reply:
Done.